css/queries.css

```diff
@@ -0,0 +1,180 @@
+/* Media queries */
+@media (min-width: 768px) {
```

Comment:
- [ ] Great job making the changes requested by the previous reviewer 💯, but, [this](https://github.com/haliljon/Portfolio-Project/pull/5#discussion_r1018174542) change is not working at all, kindly, fix the margins when the screen is in 768px. 🍨Remember, nowadays we have a lot of devices with different screen sizes, and it is very important to create responsive websites 🇶🇦 

![image](https://user-images.githubusercontent.com/91301423/201086584-6cfad042-f2a9-4384-860e-e4abfd188877.png)